[DOC-125] added diagnostics page for Voyager (#19486)
yugabyte/yugabyte-db: docs/content/preview/yugabyte-voyager/migrate/bulk-data-load.md

       --escape-char <ESCAPE_CHAR> \ # for csv format only. default: '"'
       --quote-char <QUOTE_CHAR> \ # for csv format only. default: '"'
       --has-header \ # for csv format only. default: false

>       --null-string <NULL_STRING> # default '' (empty string) for csv and '\N'  for text

Reviewer: indentation

```

Refer to [import data file](../../reference/bulk-data-load/import-data-file/) for details about the arguments.

### Load multiple files into the same table

> The import data file command also supports importing multiple files to the same table by providing the `--file-table-map` flag `<fileName>:<tableName>` entry for each file, or by passing a glob expression in place of the file name. For example, `fileName1:tableName,fileName2:tableName` or `fileName*:tableName`.

Reviewer: ```suggestion
The import data file command also supports importing multiple files to the same table by providing the `--file-table-map` flag with a `<fileName>:<tableName>` entry for each file, or by passing a glob expression in place of the file name. For example, `fileName1:tableName,fileName2:tableName` or `fileName*:tableName`.
```

### Incremental data loading

> The import data file command also supports importing files to the same table across multiple runs. For example,

Reviewer: ```suggestion
You can also import files to the same table across multiple runs. For example, you could import `orders1.csv` as follows:
```

```sh
yb-voyager import data file --file-table-map 'orders1.csv:orders' ...
```

> followed by,

Reviewer: ```suggestion
And then subsequently import `orders2.csv` to the same table as follows:
```

```sh
yb-voyager import data file --file-table-map 'orders2.csv:orders' ...
```

> If you want to import an updated version of the same file (that is, having the same file name and data-dir), use the `--start-clean` flag and proceed without truncating the table. `yb-voyager` will ingest the data present in the file in upsert mode. For example,

Reviewer: ```suggestion
To import an updated version of the same file (that is, having the same file name and data-dir), use the `--start-clean` flag and proceed without truncating the table. yb-voyager ingests the data present in the file in upsert mode. For example:
```
